docsrc/content/api.fsx

```diff
@@ -1,116 +1,103 @@
-(*** hide ***)
-// This block of code is omitted in the generated HTML documentation. Use 
-// it to define helpers that you do not want to show in the documentation.
-#I "../../bin/FSharp.Data.JsonValidation/"
-#r "../../bin/FSharp.Data.JsonValidation/FSharp.Data.JsonValidation.dll"
-#r "../../packages/FSharp.Data/lib/net40/FSharp.Data.dll"
-open FSharp.Data
```

Comment:
This file was being compile-checked. What's the value in removing it?

Reply:
Is the purpose of this file to actually be a script file to be executed or a doc of how the API works?